internal/deploy/kustomize/daily/examples/watch-all-namespaces/kustomization.yaml

```diff
@@ -3,8 +3,9 @@ kind: Kustomization
 
 namespace: rco-ns
 
-bases:
-- ../../overlays/watch-all-namespaces
 
-patchesStrategicMerge:
-- rco-ns.yaml
+resources:
+- ../../overlays/watch-all-namespaces
+patches:
+- path: role1.yaml
```

Comment:
@idlewis Thanks for the PR. It'll be good to have file names that indicate what it contains. Perhaps the name of the resource could be used. i.e. rco-leader-election-cluster-rolebinding.yaml